core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -155,11 +157,17 @@ angular.module('oppia').controller(
           return;
         }
 
-        [$scope.activeSuggestionId, $scope.activeSuggestion] = (
-          remainingSuggestions.pop());
+        [$scope.activeSuggestionId, $scope.activeContribution] = (
+          remainingContributions.pop());
+        $scope.activeSuggestion = $scope.activeContribution.suggestion;
+        $scope.activeSuggestionDetails = $scope.activeContribution.details;
```

Comment:
Should `$scope.activeSuggestionDetails` be `$scope.activeContributionDetails`? Since the details are derived from the contribution opportunity.

Reply:
done